### PR TITLE
core/manager: use cellescape and don't be too aggressive

### PR DESCRIPTION
**Pull request by YHNdnzj**

### CODE REVIEW

**Comment by YHNdnzj** [comment]
cc @evverx

**Comment by evverx** [comment]
@YHNdnzj thanks! I'm pointing a bespoke fuzzer at this whole notification thing. I'll take a look once I'm done there.

File: src/core/manager.c [hunk 1]
```diff
@@ -2586,22 +2586,17 @@ static void manager_invoke_notify_message(
...
+                char buf[CELLESCAPE_DEFAULT_LENGTH];
 
-                buf = strv_join(tags, ", ");
-                if (buf)
-                        x = ellipsize(buf, 20, 90);
-                if (x)
-                        y = cescape(x);
-
-                log_unit_debug(u, "Got notification message \"%s\", ignoring.", strnull(y));
+                log_unit_debug(u, "Got notification message from unsupported unit, ignoring: %s",
```
Comment (yuwata) [inline src/core/manager.c]: `unsupported unit` -> `unsupported unit type` or `unexpected unit type`?
But, feel free to ignore.

**Review by yuwata — APPROVED** [review]
LGTM. I did not know we have `cellescape()`.

**Review by bluca — APPROVED** [review]

**Pull request merged**